test/test.combokeys.js

```diff
@@ -186,20 +186,28 @@ describe('combokeys.bind', function () {
 
       KeyEvent.simulate('S'.charCodeAt(0), 83, ['meta'], element)
 
-      expect(spy.callCount).to.equal(1, 'callback should fire')
-      expect(spy.args[0][0]).to.be.an.instanceOf(Event, 'first argument should be Event')
-      expect(spy.args[0][0].cancelBubble).to.be.True
-      expect(spy.args[0][0].defaultPrevented).to.be.True
+      assert.strictEqual(spy.callCount, 1, 'callback should fire')
+      assert.instanceOf(spy.args[0][0], Event, 'first argument should be Event')
+      var event = spy.args[0][0]
+      if (event.preventDefault) {
+        assert.isTrue(event.defaultPrevented)
+      } else {
+        assert.isTrue(event.cancelBubble)
```

Comment:
`cancelBubble` only really turns to true in old IE, where there is no `preventDefault`.